tests/test_tui_publish_workflow_security.py

```
@@ -1437,5 +1437,193 @@ def test_stable_release_builds_and_tests_once_before_dispatching_publishers() ->
+        environment.update(
+            {
+                "GITHUB_REPOSITORY": "manaflow-ai/cmux",
+                "PATH": f"{bin_dir}:{environment['PATH']}",
+                "POLL_INTERVAL_SECONDS": "0",
+                "WAIT_TIMEOUT_SECONDS": "5",
+                "STATE": str(state),
+                "DETAIL_STATE": str(detail_state),
+            }
+        )
+        result = subprocess.run(
+            ("bash", str(script), "cmux-tui-release.yml", "cmux-tui-v1.2.3", sha, "0", "41"),
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)